newIDE/app/src/InstancesEditor/InstancePropertiesEditor/index.js

```diff
@@ -154,11 +154,6 @@ const InstancePropertiesEditor = ({
         valueType: 'boolean',
         getValue: (instance: gdInitialInstance) => instance.hasCustomSize(),
         setValue: (instance: gdInitialInstance, newValue: boolean) => {
```

Comment:
is this checkbox still useful?

Reply:
Yes, it allows to switch between the default size and the custom one.